modules/acm_sku/acm_sku.info.yml

```diff
@@ -5,11 +5,11 @@ core: 8.x
 package: Commerce
 dependencies:
   - acquia_commercemanager:acm
+  - acquia_commercemanager:acm_sku_stock
```

Comment:
This submodule shoudn't be dependency for using skus. You're also creating circular dependency, because that submodule depends on this one.

Reply:
aah, true, didn't notice this, removing it from acm_sku_stock module

Reply:
removed